compiler/modules/front.syntax/ast.bal

```diff
@@ -5,7 +5,7 @@ import wso2/nballerina.err;
 public type Position err:Position;
 
 public type ModulePart record {|
-    ImportDecl? importDecl;
+    ImportDecl[]? importDecl;
```

Comment:
Don't think we need the optional anymore.
```suggestion
    ImportDecl[] importDecl;
```

Reply:
Fixed